src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -369,6 +369,52 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::visualise( const IECore::I
 	return result;
 }
 
+void StandardLightVisualiser::spotlightParameters( const InternedString &attributeName, const IECore::ObjectVector *shaderVector, float &innerAngle, float &outerAngle, float &lensRadius )
```

Comment:
Looks like you need to delete the original private function this was copied from - I'm getting this error from Clang :

```
src/GafferSceneUI/StandardLightVisualiser.cpp:112:6: error: unused function 'spotlightParameters' [-Werror,-Wunused-function]
void spotlightParameters( InternedString metadataTarget, const IECore::CompoundData *parameters, float &innerAngle, float &outerAngle, float &lensRadius )
```